### ISSUE TITLE
autoRenew broken in 2.4

### ISSUE THREAD

**Issue by ryshep111** [okta/okta-auth-js#204]
I am unable to set the ```autoRenew``` option to false. Tracked it down to this line in ```buildConfigObject```:

```autoRenew: config.autoRenew || config.auto_renew```

This resolves to using auto_renew if autoRenew is set to false.

**Comment by swiftone** [comment]
Thank you for the error report - While we prioritize a fix, does setting both `autoRenew` and `auto_renew` to `false` work to keep you unblocked in the meantime?

**Comment by ryshep111** [comment]
Yes you are right, that is how I unblocked myself for now. Looking more closely seems like this bug may actually exist in @okta/configuration-validation

**Comment by swiftone** [comment]
Closing in favor of https://github.com/okta/okta-oidc-js/issues/425
